Monitor

dnesterenko27 opened the pull request:
- Generated a table to show relevant orders

nosovk commented on monitor/src/js/app.js:
```diff
@@ -0,0 +1,101 @@
+let headers = new Headers();
+
+const key = 'Basic SFNlcnZpY2U6TmU1bWkxdnU=';
```
Shouldn't be here

nosovk replied:
just ask user for input, and save it to session storage, but actual credentials should be removed form github history - they are public

Commit:
Tehpostach Monitor

- Generated an order table for tehpostach
- Added styles and simple authorization
